frontend/client/src/real/java/de.amosproj3.ziofa.client/RustClient.kt

```diff
@@ -141,3 +152,19 @@ fun uniffi.client.Client.initStreamFlow() = flow {
         }
     }
 }
+
+fun uniffi.client.Client.getOdexFilesFlow(pid: UInt) = flow {
+    getOdexFiles(pid).use { stream ->
+        while (true) {
+            stream.next()?.also { file -> emit(file) } ?: break
+        }
+    }
+}
+
+fun uniffi.client.Client.getSymbolFlow(odexFilePath: String) = flow {
```

Comment:
see comment to getSymbols, i think if this "has an end", we should rather return a list of symbols so that we know that we can display the UI if we received the first list. If we get a stream, i think, for example, sorting the displayed list continously would be quite inefficient, maybe we could use a flow that completes once the last symbol is received

Reply:
Flows can complete and do not have to be indefinite.

Reply:
nevermind, i oversaw the break ^^